src/api/app/components/bs_request_overview_avatars_component.rb

```diff
@@ -0,0 +1,28 @@
+class BsRequestOverviewAvatarsComponent < ApplicationComponent
+  MAXIMUM_DISPLAYED_AVATARS = 1
+
+  def initialize(review, group)
+    super
+
+    @review = review
+    @group = group
+  end
+
+  private
+
+  def avatar_objects
+    @avatar_objects ||= case @group
+                        when :by_user
+                          [User.find_by(login: @review.by_user)]
+                        when :by_group
+                          group = Group.find_by(title: @review.by_group)
+                          [group, group.users.first(MAXIMUM_DISPLAYED_AVATARS)].flatten
+                        when :by_project
+                          project = Project.find_by(name: @review.by_project)
+                          project.users.first(MAXIMUM_DISPLAYED_AVATARS)
+                        when :by_package
+                          package = Package.find_by(name: @review.by_package)
+                          package.project.users.first(MAXIMUM_DISPLAYED_AVATARS)
```

Comment:
`Package` can have users too.

Reply:
You will possibly have to limit Package/Project relationships to certain roles too.

I'm okay if you push this into the backlog.

Reply:
I'll do it in a separate PR. There are already to many things happening here 😄

Reply:
okay, where are you tracking this?

Reply:
here it is https://trello.com/c/k3YQmJaU